sips/minutes/_posts/2016-11-29-sip-minutes.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: sip-landing
+title: SIP Meeting Minutes - 29th November 2016
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+|Topic|Reviewers| Accepted/Rejected |
+| --- | --- | --- |
+| [SIP-28 and SIP-29 - Inline meta](http://docs.scala-lang.org/sips/pending/inline-meta.html) | Josh Suereth and Iulian Dragos | Pending |
+| [SIP-24 - Repeated By Name Parameters](http://docs.scala-lang.org/sips/pending/repeated-byname.html) | Heather Miller | Pending |
+| [SIP-NN:Static](https://github.com/scala/scala.github.com/pull/491/files)| Adriaan Moors | Pending |
+|[SIP-27: Trailing commas](http://docs.scala-lang.org/sips/completed/trailing-commas.html)|Eugene Burkamo| Accepted |
+
+Jorge Vicente Cantero was the Process Lead and Travis Lee was the secretary.
+
+
+## Date and Location
+The meeting took place on 29 November 2016 via Google Hangouts at EPFL in Lausanne, Switzerland as well as other locations.
+
+Minutes were taken by Travis Lee.
+
+## Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Jorge Vicente Cantero ([@jvican](https://github.com/jvican)), Process Lead
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Heather Miller ([@heathermiller](https://github.com/heathermiller)), Scala Center
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), EPFL
+* Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), EPFL
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Independent
+* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+* Dale Wijnand ([@dwijnand](https://github.com/dwijnand)), author of SIP-27
+
+
+## Proceedings
+### Opening Remarks
+
+**Jorge** We'll talk about the SIPS for META. Eugene will start.
+
+### [SIP-28 and SIP-29 - Inline meta](http://docs.scala-lang.org/sips/pending/inline-meta.html)
+
+Eugene and co have been working hard for two months on inline and META. Previously discussed new macro system with new inline and meta features. Inline provides a facility to declare methods with inline right hand side into call side (0:01:24) and meta implements compile-time function execution to do meta-programming. Martin implemented inline mechanism in Dotty. Eugene worked on macro notations. New style macros will integrate with tools. Eugene shows how it works in IntelliJ. For example, you can print the value of the parameters. Meta blocks supported by IntelliJ. So are quasi-quotes. You can also expand macros. Will greatly help debugability.
+
+Iulian says we should flesh out the Scala meta API.
+
+The spec needs to be updated based on Martin's Dotty implementation. We need to split the SIPs for the next meeting.
+
+**Conclusion** More development needed. Must answer questions on PR.
```

Comment:
This proposal needs at least another iteration to shape up and provide concrete implementation and specification details. This proposal is therefore under revision -- Eugene, the author, will gather and address more feedback and will resubmit the proposal to analysis when it's ready.